docs/cli.md

```diff
@@ -30,50 +30,75 @@ $ sematic run examples/mnist/pytorch
 Since Sematic makes sure not to include unnecessary heavy dependencies, you may
 be prompted to install those needed by the example you want to run.
 
-
 ### User settings
 
-Certain Sematic functionalities need certain settings variables to be set. For
-example, `SnowflakeTable` needs your Snowflake credentials to be able to query
-your data for you.
+Certain Sematic functionalities need certain settings variables to be set.
+These settings control which backend Server will be used when you issue
+commands, and other user-specific configurations. For example, `SnowflakeTable`
+needs your Snowflake credentials to be able to query your data for you.
 
 You can set user settings as follows:
 
 ```shell
-$ sematic settings set VAR VALUE
+$ sematic user-settings set VAR VALUE
 ```
 
-For example
+For example:
 
 ```shell
-$ sematic settings set SNOWFLAKE_USER "foobar"
+$ sematic user-settings set SNOWFLAKE_USER "foobar"
 ```
 
-Then you can check your stored settings with
+Then you can check your stored settings with:
 
 ```shell
-$ sematic settings show
-Active settings:
+$ sematic user-settings show
+Active user-settings:
 
 SNOWFLAKE_USER: foobar
 ```
 
-These settings are simply store in the `/.sematic/settings.yaml` file on
+These settings are simply stored in the `~/.sematic/settings.yaml` file on
 your machine.
 
 You can always override them at runtime with an environment variable:
 
 ```shell
-$ SNOWFLAKE_USER="notfoobar" sematic settings show
-Active settings:
+$ SNOWFLAKE_USER="notfoobar" sematic user-settings show
+Active user settings:
 
 SNOWFLAKE_USER: notfoobar
 ```
 
 If you'd like, you can even change which directory Sematic uses to hold its
 settings by setting the environment variable `SEMATIC_CONFIG_DIR`. This can
-be either a relative path (it will be treated as relative to your home directory)
-or an absolute path.
+be either a relative path (it will be treated as relative to your home
+directory) or an absolute path.
+
+### Server settings
+
+The same way that certain settings need to be kept on the user side, certain
+ones need to configure the backend Server.
+
+These can be accessed and modified just like the user settings, but by using
+the `sematic user-settings` command instead:
+
+```shell
+$ sematic server-settings show
+Active server settings:
+
+AWS_S3_BUCKET: sematic-dev
+KUBERNETES_NAMESPACE: default
+SEMATIC_AUTHORIZED_EMAIL_DOMAIN: sematic.ai
+SEMATIC_AUTHENTICATE: 'true'
+GOOGLE_OAUTH_CLIENT_ID: XXX
+GRAFANA_PANEL_URL: XXX
+
+```
+
+These settings are simply stored in the `~/.sematic/server-settings.yaml` file
```

Comment:
How about `settings.yaml` for user settings and `server.yaml` for the server?

Reply:
+1

Reply:
Done.